cpp/src/io/parquet/writer_impl.cu

```diff
@@ -792,6 +796,113 @@ void writer::impl::init_encoder_pages(hostdevice_2dvector<gpu::EncColumnChunk> &
   stream.synchronize();
 }
 
+template <typename T>
+void print(rmm::device_uvector<T> const &d_vec, std::string label = "")
```

Comment:
Should the printer stuff be moved or removed? Does not seem to belong here.

Reply:
oh yes. this is debug printing.